doc/develop/flash_debug/host-tools.rst

```diff
@@ -162,6 +162,23 @@ partition is:
 	};
 
 
+Using SAM-BA runner on Windows (Native)
+---------------------------------------
+In order to use SAM-BA runner on Windows, you need to install the bossac
+executable. A good source is from the `Arduino IDE`_.
+The bossac executable will be located in the
+``%USERPROFILE%\AppData\Local\Arduino15\packages\arduino\tools\bossac\1.9.1-arduino2\``
+directory. You can add this directory to your path, or copy the :file:`bossac.exe`
+to a directory that is in your path. You could also use the ``--bossac`` option
+to specify the location of the bossac executable, as follows:
+
+.. code-block:: console
+
+    west flash --bossac="%USERPROFILE%\AppData\Local\Arduino15\packages\arduino\tools\bossac\1.9.1-arduino2\bossac.exe" --bossac-port="COMx"
```

Comment:
All examples must use `-r bossac` argument to make sure user selects correct runner.
The examples should use original bossac version and not Arduino.